s3-import-commons/src/main/java/no/unit/nva/publication/s3imports/FilenameEventEmitter.java

```diff
@@ -71,7 +73,7 @@ public class FilenameEventEmitter implements RequestStreamHandler {
     private final S3Client s3Client;
     private final EventBridgeClient eventBridgeClient;
     private final Clock clock;
-    private Instant timestamp;
+    private Instant commonTimestampForAllEmittedEventsIndicatingTheBeginningOfTheImport;
```

Comment:
All events related to an import must share a common timestamp, so that they can be gathered together in a report folder with a date. There is little meaning in having events that have timestamps that differ a few milliseconds from one another.